README.md

```diff
@@ -1,48 +1,55 @@
-4. Respondents may vote “-1” to veto a proposal, but must provide clear reasoning and alternate approaches to resolving the problem within the two days.
-5. A vote of -0 indicates mild disagreement, but has no effect. A 0 indicates no opinion. A +0 indicates mild support, but has no effect.
+1. Proposals are written up and submitted on https://github.com/Open-EO/PSC/issues/new for discussion and voting, by any interested party, not just committee members.
+2. Proposals need to be available for review for at least two business days before a final decision can be made. Public holidays count as business days.
+3. Respondents may vote `+1` to indicate support for the proposal and a willingness to support implementation.
+4. Respondents may vote `-1` to veto a proposal, but must provide clear reasoning and alternate approaches to resolving the problem within the two days.
+5. A vote of `-0` indicates mild disagreement, but has no effect. A `0` indicates no opinion. A `+0` indicates mild support, but has no effect.
 6. Anyone may comment on proposals on the list, but only members of the Project Steering Committee’s votes will be counted.
-7. A proposal will be accepted if it receives +2 (including the proposer) and no vetoes (-1) or has not received enough votes in 14 business days.
+7. A proposal will be accepted if it receives +2 (including the proposer) and no vetoes (-1) or has not received enough votes in 10 business days.
 8. If a proposal is vetoed, and it cannot be revised to satisfy all parties, then it can be resubmitted for an override vote in which a majority of all eligible voters indicating +1 is sufficient to pass it. Note that this is a majority of all committee members, not just those who actively vote.
 9. Upon completion of discussion and voting the proposer should announce whether they are proceeding (proposal accepted) or are withdrawing their proposal (vetoed).
 10. The Chair gets a vote.
 11. The Chair is responsible for keeping track of who is a member of the Project Steering Committee.
 12. Addition and removal of members from the committee, as well as selection of a Chair should be handled as a proposal to the committee. 
 13. The Chair adjudicates in cases of disputes about voting.
 
-## When is Vote Required?
+### When is Vote Required?
 
 Generally speaking, technical decisions should be made by consensus within openEO components. However, if a decision cannot be decided by consensus of the committers involved in the component, any committer can request the decision be brought to a vote of the openEO PSC. Technical decision that would be suitable for such a process include:
 
 * Any change to PSC membership (new members, removing inactive members)
-* Anything that could cause backward compatibility issues in API, process specifications or shared software components.
+* Anything that could cause backward compatibility issues in normative specifications (e.g., API or process specifications).
```

Comment:
We have a remark below on the review process for shared software components. so it doesn't make sense to go with them through the PSC, too. So I changed to just specifications.